src/mux-analytics.brs

```diff
@@ -182,9 +189,11 @@ function muxAnalytics() as Object
   prototype.MUX_API_VERSION = "2.0"
   prototype.PLAYER_IS_FULLSCREEN = "true"
 
-  prototype.init = function(connection as Object, port as Object, appInfo as Object, systemConfig as Object, customerConfig as Object, hbt as Object, pp as Object)
-    m.connection = connection
-    m.port = port
+  prototype.init = function(appInfo as Object, systemConfig as Object, customerConfig as Object, hbt as Object, pp as Object)
+    m.httpPort = _createPort()
+    m.connection = _createConnection(m.httpPort)
+    m.httpRetries = 5
+    m.httpTimeout = 1500
```

Comment:
I'm assuming this is in milliseconds, right? If so, I think we'd want something along the lines of 5 seconds? 1.5 seems pretty short.

Reply:
Yes. Milliseconds. Happy to make this whatever you prefer.